### PR TITLE
update userguide matrix via element

### LINKED ISSUE #2361: Update join Monero core team Matrix server guide; screenshots outdated
Guide refers to 'matrix.monero.social' as the Synapse server to reference when signing up on the Matrix client of choice, when it should be raw 'monero.social' as everyone's handles and log-in credentials indicate.

It is necessary to update text and screenshots showing how to modify the hosting synapse server in order for user to sign up on Monero's Matrix server. One screenshot from desktop/web, another from Android and another from iOS showing 'monero.social' as server are needed.

Apparently it requires an e-mail for sign-up now too, it didn't before. Text to be updated accordingly as well.

### PR DESCRIPTION
**Pull request by HardenedSteel**
~~IOS screenshots are missing~~ IOS and Android GUIs are same so IOS screenshots are removed and not updated.

will close #2361

### CODE REVIEW

**Comment by plowsof** [comment]
its nice to see inline srcset however what are we doing here? future proofing -site for a time when jpgxl's are more widely supported when its going to be replaced shortly? _if_ we had translations set up we would be presenting translators with alot of raw html. replacing the screenshots with new avifs is perfectly acceptable. 

I'd much rather see your efforts and knowledge put into how we should be displaying images in astro / discussing fallbacks/image types with the new designer/dev.

**Comment by HardenedSteel** [comment]
> future proofing -site for a time when jpgxl's are more widely supported when its going to be replaced shortly?

yes, JPEG XL has more features and looks more [freer format*](https://en.wikipedia.org/wiki/AV1#Patent_claims), that's why I preferred over AVIF.

In my testings:
AVIF performing better for more lossy pictures
JPEG XL performing better for less (or lossless) pictures

there's more detailed comparison: https://cloudinary.com/blog/time_for_next_gen_codecs_to_dethrone_jpeg

**Comment by HardenedSteel** [comment]
converting to ready for review, IOS screenshots can be fixed with different PR

**Comment by plowsof** [comment]
NACK - remove the srcset busy work.

File: _i18n/en/resources/user-guides/join-monero-matrix.md [hunk 1]
```diff
@@ -1,34 +1,35 @@
...
 
 ### Setting the custom host server
 
-While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero core team to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
+While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero community members to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
 
-When signing up choose to edit your host server and type in `matrix.monero.social` as shown.
+When signing up choose to edit your host server and type in `monero.social` as shown.
```
Comment (nahuhh) [inline _i18n/en/resources/user-guides/join-monero-matrix.md:15]: Is this confirmed to be working?

Comment (HardenedSteel) [reply _i18n/en/resources/user-guides/join-monero-matrix.md:15]: I registered my own account using `monero.social`, so it should be like this for a long time.

Comment (nahuhh) [reply _i18n/en/resources/user-guides/join-monero-matrix.md:15]: I thought i did mine like that too, but recently (iirc) dan miller said that it wasnt working. 

so now i'm not sure 😅

Comment (plowsof) [reply _i18n/en/resources/user-guides/join-monero-matrix.md:15]: monero.social should be fine (but registrations are temporarily disabled due to spam)

Comment (nahuhh) [reply _i18n/en/resources/user-guides/join-monero-matrix.md:15]: Ok thanks

File: _i18n/en/resources/user-guides/join-monero-matrix.md [hunk 2]
```diff
@@ -1,34 +1,35 @@
 {% include disclaimer.html translated="no" translationOutdated="no" %}
 
-Monero developers primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
+Most Monero developers primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
```
Comment (nahuhh) [inline _i18n/en/resources/user-guides/join-monero-matrix.md:3]:
```suggestion
Monero contributors primarily communicate via IRC and Matrix channels. You can join IRC/Libera and Matrix rooms using Matrix. An open source network for secure, decentralized communication.
```

- "most" and "primarily" are redundant.
- Also may be good to switch "developers" to "contributors", as the socials are only for developers.

**Review by nahuhh — APPROVED** [review]
Would like someone to update the iPhone screenshots. I asked in monero-site matrix.

unfortunately, i dont have an ios device. Also, since element doesnt allow multiple accounts.. a user will have to logout from their matrix if they want to screenshot the setup. 

Approving anyway.

**Review by nahuhh — APPROVED** [review]

**Review by plowsof — APPROVED** [review]

File: _i18n/en/resources/user-guides/join-monero-matrix.md [hunk 3]
```diff
@@ -1,34 +1,37 @@
...
 
 ### Setting the custom host server
 
-While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero core team to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
+While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted by the Monero community members to support decentralization. If you do choose this option remember to set your custom server when logging in on a new device.
```
Comment (plowsof) [inline _i18n/en/resources/user-guides/join-monero-matrix.md:13]: only nit pick here is monero social is hosted by cypherstack. the general fund pays for this. its equivalent to saying that getmonero is run by community members

Comment (plowsof) [reply _i18n/en/resources/user-guides/join-monero-matrix.md:13]: ah yes, correcting my prev comment, cake sponsor the server, cypherstack provide 1 back end admin (https://www.getmonero.org/community/sponsorships/) aka "the Monero community members". 

docs.getmonero.org on the other hand, definitely hosted by "the monero community members"

Comment (plowsof) [reply _i18n/en/resources/user-guides/join-monero-matrix.md:13]:
```suggestion
While matrix is a federated protocol and you could use any host server for your account (including the default matrix.org server). We recommend changing your server to the one hosted / maintained by the Monero project [sponsors]({{ site.baseurl }}/community/sponsorships/. If you do choose this option remember to set your custom server when logging in on a new device.
```
thoughts?

Comment (plowsof) [reply _i18n/en/resources/user-guides/join-monero-matrix.md:13]: LGTM, thoughts @HardenedSteel?

**Pull request merged**